contracts/contracts/Forge.sol

```diff
@@ -39,7 +39,7 @@ contract Forge is IForge, Ownable {
   mapping(address => Score) internal score;
 
   mapping(uint256 => uint8) public override level; // wand tokenId -> level
-  uint32[] public override levelUpCost; // array of xp cost for upgrading to the respective levels
+  uint32[] public override levelUpCost; // array of xp cost for upgrading from the respective levels
```

Comment:
'to' feels more appropriate to me in this case, as the values in the array are the required amounts of XP to upgrade *to* the next level. Each amount represents a threshold to be reached, not a starting place, which would be more appropriate for 'from'

Reply:
Then we could perhaps change it to "to the next level"?

Or perhaps I am mistaken. I understood the comment as the cost to update to level 1 is stored in index 1, but it looks like it is actually in index 0 (right?).

Reply:
Right, for example, an array of `[2000, 2000, 2000, 2000, 2000]` represents the possibility of 5 levels, each requiring 2000 more xp to reach.

I agree "to the next level" is more clear since `levelUpCost[0]` represents the cost to get to level 1

Reply:
I know this isn't vital, but the comment threw me off when looking through the contracts as I didn't feel it lined up with the implementation in the `levelUp` function 😅